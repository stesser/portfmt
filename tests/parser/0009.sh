echo ".endif" | ${PORTFMT} -d 2>&1 | diff -L "0009.expected" -L "0009.actual" -u 0009.expected -
