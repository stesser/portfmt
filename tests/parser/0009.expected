portfmt: error building AST: could not find matching .if for .endif on line 1-2
